Source/WebKit/UIProcess/API/glib/WebKitBackForwardListItem.cpp

```diff
@@ -121,6 +121,8 @@ const gchar* webkit_back_forward_list_item_get_uri(WebKitBackForwardListItem* li
  *
  * Obtain the title of the item.
  *
+ * This function is deprecated and now always returns the empty string.
+ *
  * Returns: the page title of @list_item or %NULL
  *    when the title is empty.
  */
```

Comment:
```suggestion
 * Returns: an empty string
 *
 * Deprecated: 2.44
 */
```

The implementation should be just g_return_val_if_fail() followed by returning an empty string.

You'll also need to change the `WEBKIT_API` on the declaration to `WEBKIT_DEPRECATED` in WebKitBackForwardListItem.h.in.